playground/passwordGen.cpp

```diff
@@ -42,8 +42,8 @@ void passwords::properties()
     {
         if (57 > input[0] && input[0] > 47)
         {
-            passwordLength = stoi(input);
-            if (passwordLength < 6)
+            this->passwordLength = stoi(input);
+            if (this->passwordLength < 6)
```

Comment:
Use `this` pointer. That way everyone will know, that you're referencing the object instance's property and not just some passwordLength.